autogen.sh

```diff
@@ -130,6 +134,15 @@ case $do_autoconf,$do_git in
     do_autoconf=true;;
 esac
 
+if $require_nightly; then
+    echo "Validating Rust install ..."
+    command -v rustc >/dev/null 2>&1 || { echo >&2 "Remacs requires rust to be installed in order to build. Please install it via rustup: https://www.rustup.rs/; Aborting."; exit 1; }
```

Comment:
hmm, actually these checks are not depending on nightly/stable, should they be moved outside the `if`?